app/assets/javascripts/flash.js

```diff
@@ -0,0 +1,28 @@
+(function() {
+
+  this.flash = {
+    clear: function() {
+      var containerElement = document.getElementById('flash');
```

Comment:
minor: should we cache `var containerElement = document.getElementById('flash')`?

Reply:
Sounds legit since we are relying on the flash being a single constant element. I will update 👍